### PR TITLE
Fix Docker image existence check (normalization)

### PR DESCRIPTION
**Pull request by edgao**
NormalizationRunnerFactory got moved at some point. Update its path. Currently this check always passes for normalization images.

### CODE REVIEW

**Review by evantahler — APPROVED** [review]
I agree that you have properly changed the path to that file!

**Comment by edgao** [comment]
waiting for the check to pass and will merge once it turns green

**Pull request merged**
